dtrace-parser/src/lib.rs

```diff
@@ -110,6 +115,93 @@ impl Integer {
         format!("{prefix}int{}_t", self.width_to_c_str())
     }
 
+    pub fn to_asm_op(&self, i: u8) -> String {
```

Comment:
is this the right place for this? I think the point of this crate is to parse DTrace files..

Reply:
I agree, this should probably be moved to the `usdt-impl` crate. All the types we need to operate on from the `dtrace-parser` crate are public, so we have access to everything we need there.

Reply:
Could we also provide a bit more documentation as to the purpose of this function? What does it actually produce and how is it used?

Reply:
Moved to its own file and added documentation.